docs/developer-docs/latest/plugins/email.md

```diff
@@ -6,219 +6,119 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/plugins/email.html
+
 const emailTemplate = {
   subject: 'Welcome <%= user.firstname %>',
-  text: `Welcome on mywebsite.fr!
+  text: `Welcome to mywebsite.fr!
     Your account is now linked with: <%= user.email %>.`,
-  html: `<h1>Welcome on mywebsite.fr!</h1>
+  html: `<h1>Welcome to mywebsite.fr!</h1>
     <p>Your account is now linked with: <%= user.email %>.<p>`,
 };
 
 await strapi.plugins['email'].services.email.sendTemplatedEmail(
   {
     to: user.email,
-    // from: is not specified, so it's the defaultFrom that will be used instead
+    // from: is not specified, the defaultFrom is used.
   },
-  emailTemplate,
+    emailTemplate,
   {
     user: _.pick(user, ['username', 'email', 'firstname', 'lastname']),
   }
 );
 ```
 
-## Configure the plugin
-
-By default Strapi provides a local email system ([sendmail](https://www.npmjs.com/package/sendmail)). If you want to use a third party to send emails, you need to install the correct provider module. Otherwise you can skip this part and continue to configure your provider.
-
-Below are the providers maintained by the Strapi team:
-
-- [Amazon SES](https://www.npmjs.com/package/@strapi/provider-email-amazon-ses)
-- [Mailgun](https://www.npmjs.com/package/@strapi/provider-email-mailgun)
-- [Nodemailer](https://www.npmjs.com/package/@strapi/provider-email-nodemailer)
-- [SendGrid](https://www.npmjs.com/package/@strapi/provider-email-sendgrid)
-- [Sendmail](https://www.npmjs.com/package/@strapi/provider-email-sendmail)
-
-You can also find additional community maintained providers on [NPM](https://www.npmjs.com/).
+## Sending emails with a lifecycle hook
 
-To install a new provider run:
-
-<code-group>
-
-<code-block title="NPM">
-```sh
-npm install @strapi/provider-email-sendgrid --save
-```
-</code-block>
-
-<code-block title="YARN">
-```sh
-yarn add @strapi/provider-email-sendgrid --save
-```
-</code-block>
-
-</code-group>
-
-### Configure your provider
-
-After installing your provider you will need to add some settings in `./config/plugins.js`. If this file doesn't exists, you'll need to create it. Check the README of each provider to know what configuration settings the provider needs.
-
-::: note
-When using community providers, pass the full package name to the `provider` key (e.g. `provider: 'strapi-provider-email-mandrill'`). Only Strapi-maintained providers can use the shortcode format (e.g. `provider: 'sendmail'`).
-:::
-
-Here is an example of a configuration made for the provider [@strapi/provider-email-sendgrid](https://www.npmjs.com/package/@strapi/provider-email-sendgrid).
-
-**Path —** `./config/plugins.js`.
+ To trigger an email based on administrator actions in the admin panel use [Lifecycle hooks](/developer-docs/latest/development/backend-customization/models.md#lifecycle-hooks) and the [`send()` function](#using-the-send-function). For example, to send an email each time a new content entry is added in the Content Manager use the `afterCreate` lifecycle hook:
```

Comment:
```suggestion
 To trigger an email based on administrator actions in the admin panel use [lifecycle hooks](/developer-docs/latest/development/backend-customization/models.md#lifecycle-hooks) and the [`send()` function](#using-the-send-function). For example, to send an email each time a new content entry is added in the Content Manager use the `afterCreate` lifecycle hook:
```

I think we don't capitalize this product feature, to double-check with the Product Copy document on Notion. If we do, we'd have to update this through the whole documentation.